### ISSUE TITLE
[8193.9] Reproducible toolset crash (on specific hardware)

### ISSUE THREAD

**Issue by shadguy** [Beamdog/nwn-issues#21]
**To Reproduce**

1. Launch dev build toolset from Steam client or from BDC.
2. Click File->New->Next->Next
3. Click Area Wizard->Barrows Interior->Next->Next->Finish->Next->Finish
4. Right Click on "Areas" in the left pane Area Tree
5. Click New->Beholder Caves->Next->Next->Finish
6. Click the original area and attempt to open it

At this point, my cursor in the toolset is a churning blue circle and the toolset session is crashed.  I have to kill the program with the Windows Task Manager.  The next time I start the toolset, I do have the option to recover my work.

**Specifics**

 - Game build (tested in 8192, 8193.9, and a few builds in between).
 - I've access to 3 PCs for testing as of this writing; the issue exists only on one, the oldest:
        Running Windows 10, build 17134
        System: Acer Aspire M3910 x64 based PC
        Intel i3 CPU 550 @ 3.20 GHz, 2 Cores, 4 Logical Processors
        Graphics: AMD Radeon HD 6570 card driving dual monitors
        Driver Version: 15.200.1062.1004

The other two PCs that I've tried are:
* Dell Inspirion 3521, i3-3217U, Intel 4000 Grpahics
* Lenovo MT 81XH Flex IdeaPad [i7 10th gen] with nVidia GeForce MX230 GPU

**If needed, describe the bug**

I seem to be able to repro a crash with just a few steps in the toolset.  I backed up and deleted my .ini files from "Documents\Neverwinter Nights" and verified my override is empty before beginning.  

This issue was previously reported through servicedesk, here: https://beamdog.atlassian.net/servicedesk/customer/portal/1/NWNS-287 .  I mention sending dump files to Greg on Dec 6/2019 in that report.

If warranted, add videos/screenshots to showcase the issue.

**Comment by virusman** [comment]
I'm afraid this is going to be really hard to track down. It must be reproducible on a developer machine to be able to figure out what's happening.

**Comment by pstemarie** [comment]
I have had this happen to me BUT only if I use the Barrows tileset as the first area created. I believe it is caused by the StartLocation not being able to be placed properly in the Barrows without placing it yourself.

@Shadguy - I was able to circumvent this crash on my machine by manually placing the StartLocation in the Barrows BEFORE creating a second area.

**Comment by virusman** [comment]
Unfortunately, without being able to reliably reproduce this on _any_ hardware, bugs like this one are very unlikely to be fixed.

**Comment by shadguy** [comment]
Thanks for your replies, guys.  I've been slow to respond due to a health issue, and might continue to be slow for a while.  Things are improving though, on that front.

I've tried the 8193.12 toolset.   At first, the crash looked like it went away, but then I realized the default mode for editing areas is now to use tabs.  My previous testing was using the legacy behavior with tabs for areas defaulted to off.  If I manually close tabs, I can still reproduce the crash. 

My actual real issue is that I can't edit much in my PW without crashing after opening and closing a few .gff resources.  I can recompile the mod scripts, or use the Module Build utilities, but once I start to edit placeable  or creature or other blueprints - or areas - I seem to crash after opening AND CLOSING about 3 or 4 gff files (.utc, .utp, .uti, etc).  The change in the toolset's default to use tabbed areas helped me zero in on this detail about closing files causing an issue.

I understand it's hard to debug without being able to repro, Victor.  I'm hoping to get a new rig someday soon'ish, but I can crash thing thing consistently enough that I felt it was worth reporting.  I can always keep working in the 1.69 toolset, but I'm jealous of the fixes and features you've been adding in EE.  :)

-Dave
